server/public/styles.css

```diff
@@ -150,6 +187,24 @@ li {
   .col-posts {
     padding: 0 20%;
   }
+
+  .menu-cont {
+    width: 30%;
```

Comment:
Instead of a %, use px for your menu's width. % width is only really good for columns or making elements 100% width.